tests/jenkins.test.ps1

```diff
@@ -0,0 +1,163 @@
+Describe "Jenkins Configuration" {
+    Context "Installation Integrity" {
+        BeforeAll {
+            $jenkins = choco list -lo -r | ConvertFrom-Csv -Delimiter '|' -Header Package,Version | Where-Object Package -eq 'jenkins'
+            $service = Get-Service jenkins
+        }
+
+        It "Jenkins is installed" {
+            $Jenkins | Should -Not -BeNullOrEmpty
+        }
+
+        It "Jenkins is version '2.222.4'" {
+            $jenkins.version | Should -Be '2.222.4'
+        }
```

Comment:
I presume we're pinned to this version? I'd maybe set the expected version in `BeforeAll`, then use it here. The it block could be `Jenkins is the correct version` then it's not so much a magic string and it's only needing to be updated in one place?